CSC307Fall2018/team-too
# Added Cart Logic and Unfinished Cart Page

# Summary
Added the functionality for adding items to the cart via the `ProductInfo` page. This is stored in `localStorage` and can be accessed in the newly-created cart page, which is yet to be finishd.

# QA
To check that adding items is correct, go to any `ProductInfo` page and open up the web console, going to the `application` tab, and open up the `Local Storage` tab. Upon pressing "add to cart", verify that items are being added or updated in the cart in `Local Storage`.
